events/hmo_on_office_gained.txt

```diff
@@ -0,0 +1,167 @@
+﻿namespace = office_gain
+
+office_gain.3 = {
+	type = character_event
+	hidden = yes
+	trigger = {
+		exists = employer
+		NOT = {
+			employer = {
+				has_variable = given_innovation_country
+			}
+		}
+		has_tech_office = yes
+		OR = {
+			has_trait = obsessive
+			has_trait = intelligent
+			has_trait = polymath
+			has_trait = scholar
+			employer = {
+				is_ai = yes
+			}
+		}
+		NOT = {
+			has_variable = given_innovation_recently
+		}
+	}
+	weight_multiplier = {
+		modifier = {
+			factor = 1.25
+			has_trait = obsessive
+		}
+		modifier = {
+			factor = 1.25
+			has_trait = intelligent
+		}
+		modifier = {
+			factor = 1.25
+			has_trait = polymath
+		}
+		modifier = {
+			factor = 1.25
+			has_trait = scholar
+		}
+		modifier = {
+			factor = 5
+			employer = {
+				is_tribal = yes
+			}
+		}
+		modifier = {
+			factor = 0.8
+			employer = {
+				is_ai = yes
+			}
+		}
+	}
+	immediate = {
+		save_scope_as = innovation_giver
+		if = {
+			limit = {
+				NOT = {
+					employer = {
+						is_tribal = yes
+					}
+				}
+			}
+
+			set_variable = {
+				name = given_innovation_recently
+				days = 1825
+			}
+			employer = {
+				set_variable = {
+					name = given_innovation_country
+					days = 730
+				}
+			}
+		}
+		else = {
+			set_variable = {
+				name = given_innovation_recently
+				days = 1105
+			}
+			employer = {
+				set_variable = {
+					name = given_innovation_country
+					days = 540
+				}
+			}
+		}
+
+		employer = {
+			trigger_event = {
+				id = office_gain.4
+			}
+		}
+	}
+}
+
+office_gain.4 = {
```

Comment:
this buffs the special innovation event for tribes so that it will also increase their tech percent as well as giving an innovation.